hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/DefaultContainerExecutor.java

```diff
@@ -566,11 +566,10 @@ public void writeLocalWrapperScript(Path launchDst, Path pidFile,
   @Override
   public boolean signalContainer(ContainerSignalContext ctx)
       throws IOException {
-    String user = ctx.getUser();
     String pid = ctx.getPid();
     Signal signal = ctx.getSignal();
     LOG.debug("Sending signal {} to pid {} as user {}",
```

Comment:
unless the debug is guarded, same cost

Reply:
Sorry for my mistake. I  did not notice that developer has changed this logging call to parameterized logging and removed the logging guard.